core/src/main/java/org/sonarsource/sonarlint/core/plugin/DefaultPluginRepository.java

```diff
@@ -73,6 +82,29 @@ public void stop() {
     infosByKeys.clear();
   }
 
+  public Collection<LoadedAnalyzer> getLoadedAnalyzers() {
+    return infosByKeys.values().stream()
+      .map(this::pluginInfoToLoadedAnalyzer)
+      .collect(Collectors.toList());
+  }
+
+  private LoadedAnalyzer pluginInfoToLoadedAnalyzer(PluginInfo p) {
+    String version = p.getVersion() != null ? p.getVersion().toString() : null;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "getVersion" is nullable here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2259)